tests/app/main/test_frameworks.py

```diff
@@ -3993,7 +3993,8 @@ def test_we_abort_if_agreement_does_not_match_supplier_framework(
         supplier_framework = self.supplier_framework(framework_slug='g-cloud-8', on_framework=True)['frameworkInterest']
         return_supplier_framework.return_value = supplier_framework
 
-        res = self.client.get("/suppliers/frameworks/g-cloud-8/234/signer-details")
+        # Do request
```

Comment:
I don't think we really need `Do request` as a comment on `.get()` and `.post()`... no biggie though.